back/index.js

```diff
@@ -145,13 +146,14 @@ app.route('/profiles/:id')
 		const param = request.query.token
 		const idProfile = request.params.id
 		jwt.verify(param, secret, (err, authData) => {
-			const userId = authData.sub
+			const userEmail = authData.iss
+			console.log(authData)
 			if (err) {
 				response.sendStatus(401)
 			} else {
 				connection.query(
 					//retrieve only id 1 because we are changing database, will be updated on next PR
-					`SELECT nickname, firstname, lastname FROM profile WHERE id = 1`,
+					`SELECT * FROM profile INNER JOIN account on email = '${userEmail}'`,
```

Comment:
SELECT * raises an issue : you might get too much data from your request

Reply:
Didn't saw you message, but it should be fixed with the 2 following commits. Still a * request, but on profile table, where there are no sensitive datas like password.

Reply:
but there may be some day, so this is not recommended. please change it :)